ffc-ahwr-event/application-status-event/index.js

```diff
@@ -0,0 +1,35 @@
+const queryEntities = require('../azure-storage/query-entities')
+
+const onApplicationStatusEvent = async (context, event) => {
+  const partitionKey = `${event.properties.sbi}`
+  let rowKey = `${partitionKey}_${new Date(event.properties.action.raisedOn).getTime()}`
+  const checkIfDuplicate = await queryEntities(
+    'ffcahwrapplicationstatus',
+    partitionKey,
+    rowKey
+  )
+  if (checkIfDuplicate.length > 0) {
+    rowKey = `${event.properties.id}_${new Date().getTime()}`
+    event.properties.status = 'duplicate event'
+  }
+  const applicationStatusEvent = {
+    PartitionKey: partitionKey,
+    RowKey: rowKey,
+    Status: event.properties.status,
+    ChangedBy: event.properties.action.raisedBy,
+    ChangedOn: event.properties.action.raisedOn,
+    ApplicationStatus: event.properties.action.data.applicationStatus,
+    Remark: event.properties.action.data.remark
```

Comment:
I realise that the _Remark_ column was mentioned in the ticket but I'd remove it as I don't see it being populated given there is no where for a user to enter it within the backoffice. If it's required later then it can be added at that time, no point in having an empty column sitting in a db.

Reply:
- remark removed,
- added additional fields